packages/mobile/package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do you know why the package-lock.json changed? It doesn't look like any changes were made to package.json